src/test/java/com/ably/kafka/connect/integration/ChannelSinkConnectorIT.java

```diff
@@ -0,0 +1,61 @@
+//package com.ably.kafka.connect.integration;
```

Comment:
I assume the plan is to update this test?

Reply:
Removed, will add it in a new PR.